utopia-remix/app/routes/projects.tsx

```diff
@@ -373,6 +374,8 @@ const ProjectsHeader = React.memo(({ projects }: { projects: ProjectWithoutConte
     }
   }
 
+  const [sortMenuOpen, setSortMenuOpen] = useState(false)
```

Comment:
it would be nice to stick to `React.useState` here (and drop the `useState` import)